### ISSUE TITLE
build-x86.sh: not found

### ISSUE THREAD

**Issue by Road-Drum** [BlissRoms-x86/manifest#29]
For some reason I've been having this problem even after redo everything based on instructions. Branch used is `q-x86-12.1`. Other branch such as `q` and `q10-x86` doesn't have this problem

Also when building, build failed with `gcc: not found error` while host system already has gcc installed. Related?

Host info:
Ubuntu 19.10 
Linux 5.5.12-050512-generic

**Comment by electrikjesus** [comment]
Did you make sure you have all the dependencies installed that are mentioned in the readme?

**Comment by electrikjesus** [comment]
Also, our Q revision no longer uses the build-x86.sh command. After you '. build/envsetup.sh', you can just run 'build-x86'

**Comment by Road-Drum** [comment]
Now suddenly with `. build/envsetup.sh` it runs, which is weird since I did the same thing a while ago. I'm still seeing conflicts though

```
===========================================================================
           ALERT : Conflicts Observed while patch application !!           
===========================================================================
device/generic/common
frameworks/native

```
With manual `git am -3 <patch file>` this occurs:

```
git am -3 0005-Allow-broken-phony-targets.patch
Applying: Allow broken phony targets
error: sha1 information is lacking or useless (BoardConfig.mk).
error: could not build fake ancestor
Patch failed at 0001 Allow broken phony targets
```

now I can manually patch it, call `make -j4 iso_img` and be done with it, but I'd like to add proprietary files and use the `build-x86` script for easier future syncing and patching. Any strategy on how to deal with conflicts?

**Comment by electrikjesus** [comment]
My current process for conflicts is to take them one by one. I use 'git am <patch file>' and if that fails, I use 'patch -p1 < <patch file>' which will create the .orig & .rej files for me to see if it is really a conflict or if it's already been merged elsewhere.

**Comment by Road-Drum** [comment]
Thanks. Say that I did the same and managed to resolve it, how can I make the script to see that it has been applied? Many times I perform it manually but the script still can't see that it has been applied. Some suggest to `git add <filename>` the manual patch but this still doesn't seem to resolve it.

**Comment by Road-Drum** [comment]
Also some of the patch says

```
patch -p1 --merge < 0003-HDMI-FIX-drm-i915-delay-hotplug-scheduling.patch 
patching file drivers/gpu/drm/i915/i915_drv.h
Hunk #1 already applied at 145.
can't find file to patch at input line 28
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff --git a/drivers/gpu/drm/i915/intel_hotplug.c b/drivers/gpu/drm/i915/intel_hotplug.c
|index 9a8018130237..b4f5d372d6e7 100644
|--- a/drivers/gpu/drm/i915/intel_hotplug.c
|+++ b/drivers/gpu/drm/i915/intel_hotplug.c
```

I guess there are a lot of yet-to-be upstreamed sources given to this error. manual inspection of `k5.6rc6_si_ylng` shows `intel_hotplug.c` simply isn't there. the earlier error in `patch -p1 --merge` and `git am -3` shows that the patch is trying to patch target file with different object hashes hence the error.

all of this happens on `q10-x86` manifest. I've another directory for `q-x86-12.1` which also contains patch conflicts but I'll see if it's caused by hash mismatch or not. I'll probably make another directory and see if `q` fails to patch too.

**Comment by electrikjesus** [comment]
The kernel patches I have added in there are for Kernel 4.19, not 5.x. So I wouldn't expect those to apply clean

**Comment by Road-Drum** [comment]
> The kernel patches I have added in there are for Kernel 4.19, not 5.x. So I wouldn't expect those to apply clean

I've noticed there are blissos-yantra iso builds with `k5.6rc6_si_ylng`. which branch of `BR-x86` contains patches for `k5.6rc6_si_ylng`? by default the `q10-x86` uses kernel `k5.6rc6_si_ylng`

anyway I'm building using `q-x86-12.1` branch since I managed to resolve the conflicts. still in progress but I'd like to use newer blissos and kernel since my laptop (Ryzen 5 2500U) prefers 5.3+ kernel to do things properly
